tests/mail/tests.py

```diff
@@ -604,6 +605,35 @@ def test_sanitize_address(self):
             '=?utf-8?q?T=C3=B3_Example?= <=?utf-8?b?dMOz?=@example.com>'
         )
 
+@unittest.skipIf(sys.platform.startswith('win'), "Windows tests would require non-UTC Jenkins server.")
+class MailTimeZoneTests(HeadersCheckMixin, TestCase):
+    """
```

Comment:
`# setting the timezone requires a database query on PostgreSQL.` as in 37aec6b186941566a664369e031d72b2ddd204fc would be sufficient for the comment
We try to avoid ticket references except for obscure issues.